### ISSUE TITLE
Fluid monitor cover show weird level when enable overflow voiding.

### ISSUE THREAD

**Issue by lordIcocain** [GTNewHorizons/GT-New-Horizons-Modpack#14105]
### Your GTNH Discord Username

lord_cocain

### Your Pack Version

2.3.5

### Your Server

private server

### Java Version

Java 19

### Type of Server

Vanilla Forge

### Your Expectation

Fluid monitor cover show correct level in any tank mode. like at image
<img width="316" alt="image" src="https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/62835225/c1bb6184-c0b9-4c41-80b2-eaf0da318146">

### The Reality

What is show when overflow enabled.
<img width="313" alt="image" src="https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/62835225/2e51381e-0d73-42b9-b4b3-59b62b38a89a">

### Your Proposal

Should show correct fluid level in overflow mode too.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by boubou19** [comment]
is it still happening on 2.3.6?

**Comment by lordIcocain** [comment]
Yes

**Comment by chochem** [comment]
basically the same issues as here https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11519
TLDR: overflow voiding is a bit hacky and sets the internal storage to intmax.

Waila was fixed but I think the portable scanner still shows intmax as well.
